### PR TITLE
Fix: assorted minor J-Link fixes

### PR DESCRIPTION
**Pull request by perigoso**
## Detailed description

Not much to note, a couple of fix-ups done while working on other features/branches

## Your checklist for this pull request

* [x] I've read the [Code of Conduct](https://github.com/blackmagic-debug/blackmagic/blob/main/CODE_OF_CONDUCT.md)
* [x] I've read the [guidelines for contributing](https://github.com/blackmagic-debug/blackmagic/blob/main/CONTRIBUTING.md) to this repository
* [x] It builds for hardware native (`make PROBE_HOST=native`)
* [x] It builds as BMDA (`make PROBE_HOST=hosted`)
* [x] I've tested it to the best of my ability
* [x] My commit messages provide a useful short description of what the commits do

### CODE REVIEW

**Review by dragonmux — CHANGES_REQUESTED** [review]
Couple of review items, but this looks good to us. What sort of urgency do you see behind these fixes landing - are they in your opinion show-stoppers for v1.10, or can they be deferred to v2.0?

It's a touch difficult to tell the severity of what's being fixed, though we're leaning toward v1.10 given the docs fixes.

File: src/platforms/hosted/jlink.c [hunk 1]
```diff
@@ -550,7 +550,7 @@ static bool jlink_kickstart_power(void)
 {
 	if (!(jlink.capabilities & JLINK_CAPABILITY_POWER_STATE)) {
 		if (jlink.capabilities & JLINK_CAPABILITY_KICKSTART_POWER)
-			DEBUG_ERROR("J-Link does not support JLINK_CMD_POWER_GET_STATE command, but does support kickstart power"
+			DEBUG_ERROR("J-Link power state command is not available, but kickstart power is"
 						", this is unexpected\n");
```
[the remark this answers is not in the document]
Comment (perigoso) [reply src/platforms/hosted/jlink.c:554]: Looks like it not longer results in a too long line, fixed!

**Comment by perigoso** [comment]
> Couple of review items, but this looks good to us. What sort of urgency do you see behind these fixes landing - are they in your opinion show-stoppers for v1.10, or can they be deferred to v2.0?
> 
> It's a touch difficult to tell the severity of what's being fixed, though we're leaning toward v1.10 given the docs fixes.

These are very low priority fixes, it can be deferred to v2.0, it should also be very low impact though, as it is in essence just a couple string changes.

**Review by dragonmux — APPROVED** [review]
LGTM, we'll merge this when the v2.0 cycle starts then.

File: README.md [hunk 2]
```diff
@@ -84,10 +84,10 @@ to connect.
...
 * Black Magic Probe firmware probes via the USB-serial port
-* ST-LinkV2 and V3 with recent firmware
+* ST-Link v2, v2.1 and v3 with recent firmware
 * CMSIS-DAP compatible probes
-* JLINK probes
-* FTDI MPSSE based probe.
+* J-Link probes
+* FTDI MPSSE based probes
 
 When connected to a single BMP supported probe, starting "blackmagic" w/o any
```
Comment (dragonmux) [inline README.md:92]: Some additional feedback we've had suggested in private, is to switch the quotes over to backticks for the binary name

Comment (perigoso) [reply README.md]: Thats a good sugestion, done.

**Review by dragonmux — APPROVED** [review]

**Pull request merged**